### PR TITLE
fix(clinical details): now the other fields arent added for no reason…

### PR DESCRIPTION
**Pull request by MayaSasson**
… and delete extra info

### CODE REVIEW

File: client/src/components/App/Content/InvestigationForm/TabManagement/ClinicalDetails/useClinicalDetails.ts [hunk 1]
```diff
@@ -198,13 +196,11 @@ const useClinicalDetails = (parameters: useClinicalDetailsIncome): useClinicalDe
 
     const getSymptomsList = (patientInvestigation: any) => {
         const symptoms: string[] = patientInvestigation.investigatedPatientSymptomsByInvestigationId.nodes.map((symptom: any) => symptom.symptomName);
-        if (patientInvestigation.otherSymptomsMoreInfo) symptoms.push(otherSymptomFieldName);
```
Comment (NaamanTsur) [inline client/src/components/App/Content/InvestigationForm/TabManagement/ClinicalDetails/useClinicalDetails.ts:201]: Why were these lines removed?

Comment (MayaSasson) [reply client/src/components/App/Content/InvestigationForm/TabManagement/ClinicalDetails/useClinicalDetails.ts:201]: Because other is already in the db, so there is no need in adding it

**Review by NaamanTsur — DISMISSED** [review]

File: client/src/components/App/Content/InvestigationForm/TabManagement/ClinicalDetails/ClinicalDetails.tsx [hunk 2]
```diff
@@ -165,6 +165,13 @@ const ClinicalDetails: React.FC<Props> = ({ id, onSubmit }: Props): JSX.Element
             setValue(ClinicalDetailsFields.OTHER_SYMPTOMS_MORE_INFO, '');
         }
     }, [watchDoesHaveSymptoms]);
+    
+
```
Comment (roymaniv) [inline client/src/components/App/Content/InvestigationForm/TabManagement/ClinicalDetails/ClinicalDetails.tsx:169]: שורת רווח מיותרת

File: client/src/commons/Forms/PlacesTypesAndSubTypes/PlacesTypesAndSubTypes.tsx [hunk 3]
```diff
@@ -59,7 +59,7 @@ const PlacesTypesAndSubTypes: React.FC<Props> = (props: Props): JSX.Element => {
     const handleSubTypeInputChange = (subTypeInput: string) => {
         setPlaceSubTypeInput(subTypeInput);
         if (subTypeInput === '') {
-            onPlaceSubTypeChange(placesSubTypesByTypes[placeType][0] as PlaceSubType);
+            onPlaceSubTypeChange({id: -1, displayName: ''});
```
Comment (roymaniv) [inline client/src/commons/Forms/PlacesTypesAndSubTypes/PlacesTypesAndSubTypes.tsx:62]: default object in const

**Review by roymaniv — APPROVED** [review]

**Review by EmilyLip — APPROVED** [review]

**Review by daniel7764 — APPROVED** [review]
